librosa/core/constantq.py

```diff
@@ -172,11 +175,14 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
         fmin = note_to_hz('C1')
 
     if tuning is None:
-        tuning = estimate_tuning(y=y, sr=sr)
+        tuning = estimate_tuning(y=y, sr=sr, bins_per_octave=bins_per_octave)
+
+    # Apply tuning correction
+    fmin = fmin * 2.0**(tuning / bins_per_octave)
```

Comment:
There is nothing preventing the user to set `tuning` outside the recommended range `[-0.5, 0.5)`. Bug or feature?

Reply:
Indeed.  Per your point below, I guess there isn't really a good reason to require this limit.